akka-http-core/src/main/scala/akka/http/scaladsl/model/ContentType.scala

```diff
@@ -11,13 +11,12 @@ import akka.http.javadsl.{ model ⇒ jm }
 import akka.http.impl.util.JavaMapping.Implicits._
 
 final case class ContentTypeRange(mediaRange: MediaRange, charsetRange: HttpCharsetRange) extends jm.ContentTypeRange with ValueRenderable {
-  def matches(contentType: jm.ContentType) = {
```

Comment:
don't want to start wars here, but I've grown to _really_ like braces around anything that's multi-line, since otherwise a stray empty line would suddenly cause the rest to become constructor statements...